src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
I'm completely unaware of any "faster simulators" (does that include FlightGear?). Please explain the architecture or point me to adequate documentation on the system design.